src/java-runtime/java/mono/android/Runtime.java

```diff
@@ -11,7 +11,7 @@ private Runtime ()
 	{
 	}
 
-	public static native void init (String lang, String[] runtimeApks, String runtimeDataDir, String[] appDirs, ClassLoader loader, String[] externalStorageDirs, String[] assemblies, String packageName, int apiLevel, String[] environmentVariables);
+	public static native void init (String lang, String[] runtimeApks, String runtimeDataDir, String[] appDirs, ClassLoader loader, String[] externalStorageDirs, String[] assemblies, int apiLevel);
```

Comment:
Perhaps we should *avoid* this API signature change *for now* so that we don't break the Designer?  There shouldn't be any harm in passing along "extra unused" parameters (and/or we'll find out we *do* need them!).